app/models/user.rb

```diff
@@ -0,0 +1,15 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] For this project we want users to be able to confirm their emails during registration, therefore kindly ensure to enable conifrmable, as informed in the previous review, like below:
```ruby
	devise :database_authenticatable, :registerable,
				:recoverable, :rememberable, :validatable, :confirmable
````
You can notice that confirmable has been disabled on _line 3_ as it's currently commented out, by default